tools/sdk-ai-bots/azure-sdk-qa-bot-backend/tsp/model.tsp

```diff
@@ -0,0 +1,234 @@
+import "@typespec/http";
+import "@typespec/versioning";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+namespace AzureSDKQABot;
```

Comment:
Shall we use AzureSDK.QAAssistant?

Reply:
Suggest use QAAssistant instead of bot in the spec and it would go beyond just support chatbot.